.yamato/config/universal_perf_boatattack.metafile

```diff
@@ -1,5 +1,6 @@
 project:
   name: URP_Performance_BoatAttack
+  repo: BoatAttack
```

Comment:
Do we use this somewhere? Just wondering, but it's not a blocker

Reply:
oh I didn't realise I even changed that, thank you